Avatar/avatar.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Alert</title>
+    <link rel="stylesheet" href="avatar.css">
+    <script src="https://kit.fontawesome.com/60d2206e29.js" crossorigin="anonymous"></script>
```

Comment:
Include sctipt tag either in bottom or use defer

Reply:
will fix it